dss.go

```diff
@@ -0,0 +1,123 @@
+package globalsign
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/hex"
+	"strings"
+	"time"
+)
+
+// DSSService .
+type DSSService interface {
+	Login(*LoginRequest) (*LoginResponse, error)
+	Identity(*IdentityRequest) (*IdentityResponse, error)
+	Timestamp(*TimestampRequest) (*TimestampResponse, error)
+	Sign(*SigningRequest) (*SigningResponse, error)
+	CertificatePath() (*CertificatePathResponse, error)
+	TrustChain() (*TrustChainResponse, error)
+	// DSS Identity and sign process.
+	DSSGetIdentity(context.Context, string, *IdentityRequest) (*DSSIdentity, error)
+	DSSIdentitySign(context.Context, string, *IdentityRequest, []byte) ([]byte, error)
+	DSSIdentityTimestamp(context.Context, string, *IdentityRequest, []byte) ([]byte, error)
+}
+
+// DSSIdentity represent acquired credential
```

Comment:
represents, maybe fix the line length etc?

Reply:
Updated at a3ddc191292d0952b56e2269c3cbfa6f517b98fa